packages/unmock-cli/src/commands/init.starter.tmp.ts

```diff
@@ -0,0 +1,56 @@
+export default `\
+/**
+ * Unmock Starter Template
+ *
+ * This is just to get you started, feel free to delete this.
+ * Place your tests in this file.
+ */
+const unmock = require("unmock");
```

Comment:
In future, we could also support creating TS templates with ES6 imports etc.

Reply:
Perhaps we could have a flag for it: `unmock init --useTs`